project.godot

```diff
@@ -8,9 +8,14 @@
 
 config_version=4
 
-_global_script_classes=[  ]
```

Comment:
not really sure what happened here... is this because some classes refer to Villager? e.g. here - https://github.com/marc-hugh-tom/game-off-2020/pull/2/files#diff-58cd9121346ff042724799d1de5f3c56380647ea8967c38c50e6df87013413f2R3

Reply:
Answer seems to be that Godot registers a mapping so you can just refer to "Villager" and not "/path/Villager.gd": https://www.reddit.com/r/godot/comments/b4bzm3/what_is_the_meaning_of_config_version_in_a/